builtin/method_str.py

```diff
@@ -130,3 +135,148 @@ def Call(self, rd):
             return value.Null
 
         return RegexMatch(string, indices, capture)
+
+
+class Replace(vm._Callable):
+
+    def __init__(self, mem, expr_ev):
+        # type: (state.Mem, expr_eval.ExprEvaluator) -> None
+        self.mem = mem
+        self.expr_ev = expr_ev
+
+    def EvalSubstExpr(self, expr, blame_loc):
+        # type: (value.Expr, loc_t) -> str
+        res = self.expr_ev.EvalExpr(expr.e, blame_loc)
+        if res.tag() == value_e.Str:
+            return cast(value.Str, res).s
+
+        raise error.TypeErr(res, "expected expr to eval to a string",
```

Comment:
How about writing this "expected Expr to evaluate to Str" to be consistent with how we spell the types in other cases?